frontend/src/components/entryboard/Entryboard.vue

```diff
@@ -0,0 +1,86 @@
+<script setup lang="ts">
+import { reactive } from 'vue';
+import LockerRoomIdentifier from './LockerRoomIdentifier.vue';
+import TeamInfo from './TeamInfo.vue';
+import EditButton from './EditButton.vue';
+import placeholderImage from "@/assets/img/hockeyClipart.jpg";
+
+
+interface Team {
+  id: number
+  name: string,
+  image?: HTMLImageElement
+}
+
+function getPlaceholderImage(): HTMLImageElement {
+  var i = new Image();
+  i.alt = "team logo";
+  i.src = placeholderImage;
+  i.className = "team-image";
+  return i;
+}
+
+function getTeams() : Team[] {
+  //TODO: This function should ask the backend for the data.
```

Comment:
Thanks for putting a TODO here. That will help to track down places that need updating later.